lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationClient.h

```diff
@@ -218,7 +218,12 @@ class GDBRemoteCommunicationClient : public GDBRemoteClientBase {
 
   bool GetpPacketSupported(lldb::tid_t tid);
 
-  bool GetxPacketSupported();
+  enum class xPacketState {
+    Unimplemented,
+    Prefixed, // Successful responses start with a 'b' character.
+    Bare,     // No prefix, packets starts with the memory being read.
```

Comment:
I would add something to these to indicate which is/was gdb/lldb's style of packet.